lnrpc/README.md

```diff
@@ -183,3 +183,65 @@ The following commands are available with `make`:
 * `rpc-check`: Runs both previous commands and makes sure the git work tree is
   not dirty. This can be used to check that the `.proto` files are formatted
   and compiled properly.
+
+## Protocol buffers and plugins
+
+First, you need protoc. It's generic gcc thing so install `build-essential`
```

Comment:
Should we mention here that this is only necessary if the above mentioned dockerized `make` commands cannot be used (for whatever reason)?

Reply:
And just out of curiosity, why weren't you able to use those command? Anything we can add to make it easier for people to use them? Since installing all tools manually sounds like a lot of work and those versions will eventually fall out of sync as well I assume (people forget to update READMEs when bumping version numbers in script files).

Reply:
I'd be in favor of writing scripts to automate them so they are more visible to programmers.

Anyone who is updating the versions of those tools would be remiss to not update related documentation.

Reply:
Again, that's the whole point of having everything automated as a Docker script, that way the versions are all pulled directly from the `go.mod` where possible. Again my question: What made it impossible or impractical for you to use the Docker scripts (`make rpc`) and go down the manual route that lead to this description in the README?